akka-cluster/src/main/resources/reference.conf

```diff
@@ -34,6 +34,11 @@ akka {
     # in case of a network partition the persistent actors in the non-surviving partitions
     # must be stopped before corresponding persistent actors are started somewhere else.
     down-removal-margin = 20s
+
+    # Allow new members to join the cluster when there is no convergence.
```

Comment:
strictly speaking it's already possible to join (status = Joining), so we should try to find a better description and property name